### DISCUSSION TITLE
how fix to

### DISCUSSION THREAD

**Discussion by AbdalaMask** [unoplatform/uno#4972] [Q&A]
![image](https://user-images.githubusercontent.com/37013638/104819660-9cad9800-5837-11eb-81e1-b883dfa6d976.png)

**Comment by AbdalaMask** [comment]
RadialGradientBrush not using to uap

**Comment by jeromelaban** [comment]
Thanks for the report! 

You will need to add WinUI 2.5 Library: https://docs.microsoft.com/en-us/windows/apps/winui/winui2/getting-started to access this class on the UWP head.

<blockquote><img src="https://docs.microsoft.com/en-us/media/logos/logo-ms-social.png" width="48" align="right"><div><strong><a href="https://docs.microsoft.com/en-us/windows/apps/winui/winui2/getting-started">Getting started with the Windows UI library - WinUI</a></strong></div><div>How to install and use the Windows UI Library.</div></blockquote>

**Comment by AbdalaMask** [comment]
When you add it to a project Uno 

Gets wrong

How to use it through a library Uno
Thanks for replying

**Comment by jeromelaban** [comment]
Thanks for your answer.

Can you provide a sample project (using the default visual studio Uno templates) that reproduces the issue ? Thanks!

**Comment by AbdalaMask** [comment]
[repos.zip](https://github.com/unoplatform/uno/files/5827987/repos.zip)
this

**Comment by AbdalaMask** [comment]
is fixed

**Comment by AbdalaMask** [comment]
Has the problem been found?
jeromelaban

**Comment by AbdalaMask** [comment]
new Error
![image](https://user-images.githubusercontent.com/37013638/105225340-18c11c00-5b67-11eb-906c-3f4d3f3d42f7.png)

**Comment by AbdalaMask** [comment]
new error 
![image](https://user-images.githubusercontent.com/37013638/105229269-596f6400-5b6c-11eb-9cda-5124e557e463.png)

**Reply by AbdalaMask** [reply]
https://github.com/AbdalaMask/EVStudio.Scada.Platforms
